docs/integrations/fastapi.md

```diff
@@ -4,21 +4,153 @@ title: FastAPI
 
 # FastAPI
 
-Strawberry provides support for [FastAPI](https://fastapi.tiangolo.com/) with the ASGI integration.
+Strawberry provides support for [FastAPI](https://fastapi.tiangolo.com/) with a custom [APIRouter](https://fastapi.tiangolo.com/tutorial/bigger-applications/#apirouter) called `GraphQLRouter`.
 
 See below example for integrating FastAPI with Strawberry:
 
 ```python
 from fastapi import FastAPI
-from strawberry.asgi import GraphQL
+from strawberry.fastapi import GraphQL
 
 from api.schema import schema
 
 graphql_app = GraphQL(schema)
```

Comment:
Shouldn't all references to `GraphQL` be `GraphQLRouter` instead, both here and in `RELEASE.md`?

Reply:
Yes, I'm updating docs according to latest code changes and to make them working when copie. And I'm make code examples in docs to run as are, without changes.